toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -232,13 +232,13 @@ func (c *Chroot) Initialize(tarPath string, extraDirectories []string, extraMoun
 				// Best effort cleanup in case mountpoint creation failed mid-way through. We will not try again so treat as final attempt.
 				cleanupErr := c.unmountAndRemove(leaveChrootOnDisk, unmountTypeLazy)
 				if cleanupErr != nil {
-					logger.Log.Warnf("Failed to cleanup chroot (%s) during failed initialization. Error: %s", c.rootDir, cleanupErr)
```

Comment:
You don't need to change this code. It is fine if cleanup errors just report warnings instead of adding to the error.